### PR TITLE
Export Tables as Excel and PDF

### PR DESCRIPTION
**Pull request by AbdulSir**
This PR references [AP-30](https://soen390-team12.atlassian.net/browse/AP-30?atlOrigin=eyJpIjoiNGUwZjYzNTkwN2FmNGM5NjlmZmI0ZjFjZjIyZjJmZGEiLCJwIjoiaiJ9) user story about exporting PDF and Excel

### CODE REVIEW

**Review by purplesmurf1998 — CHANGES_REQUESTED** [review]
Test the functionalities. Downloading excel works fine, however printing to PDF cuts the table off, any way of fixing this?

File: frontend/src/views/pages/inventory/InventoryTable.vue [hunk 1]
```diff
@@ -50,10 +52,23 @@
...
+//import { fetchList } from '@/api/article'
+import { parseTime } from '@/utils'
+import Print from 'vue-print-nb'
+import XLSX from 'xlsx';
+
+// Global instruction
+Vue.use(Print);
+directives: {
+    print
+}
```
[the remark this answers is not in the document]
Comment (AbdulSir) [reply frontend/src/views/pages/inventory/InventoryTable.vue]: We followed online instructions to print to PDF, and they had that code. However, I've tested the code without this directives and it works nicely. So it was removed in this [commit](https://github.com/purplesmurf1998/Autotracks/pull/65/commits/df58a9c09caad400c19ad64c07200d9044b06b0c)

File: frontend/src/views/pages/inventory/InventoryTable.vue [hunk 2]
```diff
@@ -50,10 +52,23 @@
...
 
-<script>
+<script >
 const axios = require("axios");
 import InventorySlot from "./InventorySlot.vue"
 import Vehicle from "../vehicle/Vehicle.vue"
+import Vue from 'vue'
+import excel from 'vue-excel-export'
+//import { fetchList } from '@/api/article'
+import { parseTime } from '@/utils'
```
[the remark this answers is not in the document]
Comment (AbdulSir) [reply frontend/src/views/pages/inventory/InventoryTable.vue]: This was deleted in this [commit](https://github.com/purplesmurf1998/Autotracks/pull/65/commits/df58a9c09caad400c19ad64c07200d9044b06b0c)

**Comment by AbdulSir** [comment]
> Test the functionalities. Downloading excel works fine, however printing to PDF cuts the table off, any way of fixing this?

Yes, we've noticed this issue with printing PDF. However, there doesn't seem a solution around it for now. Also, we can bring this issue to the PO because we can already print or save as pdf from the xlsx file. So this entire functionality can be removed.

**Review by TZnoj — APPROVED** [review]
I've read through the thread, tested the code and read through the code. Everything works and is up to standards. I'm hereby approving this branch for merge.

**Review by purplesmurf1998 — APPROVED** [review]
Will approve for now. Depending on the PO we might have to modify this functionality in the future.

**Pull request merged**